multiversx/mx-template-service
# added missing sdk-nestjs-cli dep

## Type


## Problem setting
- the `sdk-nestjs-cli` dependency was missing, causing `npm run generate-types` to fail

## Proposed Changes
- added missing dependency

## How to test
- clone this branch
- run `npm install`
- run `npm run generate-types` which should work
